src/libraries/Common/src/System/Security/Cryptography/PqcBlobHelpers.MLKem.cs

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+using KeyBlobMagicNumber = Interop.BCrypt.KeyBlobMagicNumber;
+using KeyBlobType = Interop.BCrypt.KeyBlobType;
+
+namespace System.Security.Cryptography
+{
+    internal static partial class PqcBlobHelpers
```

Comment:
This is split in to a partial for now because the original one has ML-DSA in it, but `MLDsaAlgorithm` has not been made available to Microsoft.Bcl.Cryptography, yet. When MLDsaAlgorithm is available in the M.B.C we can unify the files again.